### PR TITLE
Merge blog into dev

### PR DESCRIPTION
**Pull request by IlanHopti**

### CODE REVIEW

File: client/src/pages/blog/Blog.js
Comment (N0ku) [inline client/src/pages/blog/Blog.js:1]: You should remove all commented lines of code who are not used

File: client/src/pages/blog/Tuto.js
Comment (N0ku) [inline client/src/pages/blog/Tuto.js:1]: Also here too

File: server/blog_back.js
Comment (N0ku) [inline server/blog_back.js:1]: delete comments lines

File: client/src/components/blog/NewTuto.js
Comment (N0ku) [inline client/src/components/blog/NewTuto.js:1]: I think you can delete this file if it's empty

**Review by N0ku — CHANGES_REQUESTED** [review]
I let you somes comment where I see that changes can be made. It's just some comments and file that needs to be deleted

**Review by IlanHopti — COMMENTED** [review]
I have removed commented code

**Pull request merged**